homeassistant/components/lightwave.py

```diff
@@ -0,0 +1,48 @@
+"""
+Support for device connected via Lightwave WiFi-link hub.
+
+For more details about this platform, please refer to the documentation at
+https://home-assistant.io/components/lightwave/
+"""
+import voluptuous as vol
+
+from homeassistant.const import (
+    CONF_HOST, CONF_LIGHTS, CONF_NAME, CONF_SWITCHES)
+from homeassistant.helpers import config_validation as cv
+from homeassistant.helpers.discovery import async_load_platform
+
+REQUIREMENTS = ['lightwave==0.15']
+LIGHTWAVE_LINK = 'lightwave_link'
+DOMAIN = 'lightwave'
+
+CONFIG_SCHEMA = vol.Schema({
+    DOMAIN: vol.Schema({
+        vol.Required(CONF_HOST): cv.string,
+        vol.Optional(CONF_LIGHTS, default={}): {
```

Comment:
It's a little bit weird that we can set up the component without any devices configured. Could we improve this? Maybe require at least either lights or switches?

Reply:
Agree - I would say user error but can address it.

Reply:
Use `cv.has_at_least_one_key`.